Luminary099/P12.agc

```diff
@@ -200,7 +200,7 @@ P12INIT		DLOAD			# INITIALIZE ENGINE DATA.  USED FOR P12 AND
 COMMINIT	DLOAD	DAD		# INITIALIZE TARGET DATA.  USED BY P12, P70
 			HINJECT		# AND P71 IF IT DOES NOT FOLLOW P70.
 			/LAND/
-		STODL	RCO
+		STODL	RCD
```

Comment:
Definitely looks like `RCD`, will need to check the following Luminary099 files that define/reference the variable.

- ASCENT_GUIDANCE.agc L221, L222 and L576
- ERASABLE_ASSIGNMENT.agc L2614 (the variable definition) and L2615

Reply:
From the high-res scans this is supposed to be `RCO` and not `RCD`, the O and D characters are incredible difficult to distinguish when not printed correctly.

ERASABLE_ASSIGNMENT variable definition: 
![xnviewmp_37D2qtDkTT](https://user-images.githubusercontent.com/3440094/83530191-4af01180-a4e3-11ea-9b99-2a49d31d5b0c.png)

Luminary210 with the same variable definition
![image](https://user-images.githubusercontent.com/3440094/83530904-44ae6500-a4e4-11ea-9f01-b3bce3a5c246.png)

Reply:
Done!